src/shared/common/components/__tests__/__snapshots__/workstack.spec.jsx.snap

```diff
@@ -357,49 +357,49 @@ exports[`Workstack component should render with filtering 1`] = `
               "dataAdapter": null,
               "dataValueKey": "caseReference",
               "displayName": "Reference",
-              "isFilterable": true,
+              "filterable": true,
               "renderer": "caseLink",
             },
             Object {
               "dataAdapter": null,
               "dataValueKey": "fullName",
               "displayName": "Full Name",
-              "isFilterable": true,
+              "filterable": true,
```

Comment:
I suspect these issues were caused by return the entity 'WorkstackColumn' within 'WorkstackTypeDto' rather than 'WorkstackColumnDto'. I think once we apply the with within info service we can revert these changes here.